allennlp/modules/augmented_lstm.py

```diff
@@ -67,6 +70,19 @@ def __init__(self,
         else:
             self.input_linearity = torch.nn.Linear(input_size, 4 * hidden_size, bias=True)
             self.state_linearity = torch.nn.Linear(hidden_size, 4 * hidden_size, bias=True)
+        self.reset_parameters()
+
+    def reset_parameters(self):
+        # Use sensible default initializations for parameters.
+        block_orthogonal(self.input_linearity.weight.data, [self.hidden_size, self.input_size])
+        block_orthogonal(self.state_linearity.weight.data, [self.hidden_size, self.hidden_size])
```

Comment:
Note that the Keras default is to do orthogonal init for the "recurrent kernel" (the hidden x hidden, second one here I think), while using glorot_uniform for the other. Maybe a better default?